flake.nix

```diff
@@ -2,9 +2,14 @@
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
     flake-utils.url = "github:numtide/flake-utils";
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
+
   };
 
-  outputs = { self, nixpkgs, flake-utils }: {
+  outputs = { self, nixpkgs, flake-utils,flake-compat }: {
```

Comment:
You should run nix-fmt after. Weird whitespace here

Reply:
yes and i thought about it but usually you intigrate nix fmt in the flake but because of using flake utils over a native alternative it's not that easy as far as i understand it.